<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>603</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Simple Audio Player</string>
  </property>
  <widget class="QTableView" name="playlistView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>38</y>
     <width>381</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="currentTrack">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>9</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="savedtableView">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>38</y>
     <width>251</width>
     <height>531</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QToolButton" name="btn_add">
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_previous">
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/skip-previous.png</normaloff>:/buttons/skip-previous.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_play">
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_pause">
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/pause.png</normaloff>:/buttons/pause.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_stop">
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_next">
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/skip-next.png</normaloff>:/buttons/skip-next.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Delete_Song">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete song</string>
   </property>
  </widget>
  <widget class="QLabel" name="all_time">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>570</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="now_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>570</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>570</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="playlist_add_button">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>10</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Playlists</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playlist_save_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Save current playlist</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>42</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Genre</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_author">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_year">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_genre">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>83</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>Thames:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="red_themes">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>red circle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gray_themes">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>kiss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="green_themes">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>default</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_to_cur">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>570</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Add to current playlist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>570</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
